e2e/pages/events/enterAllocationDecisionEvent.page.js

```diff
@@ -3,6 +3,7 @@ const { I } = inject();
 const allocationDecisionRadioGroup = '#allocationDecision_proposal';
 const allocationDecisionRadioPrefix = 'allocationDecision_proposal-';
 const proposalReason = '#allocationDecision_proposalReason';
+const judgeLevelRadioNo = '#allocationDecision_judgeLevelRadio-';
```

Comment:
what does the No at the end mean?

Reply:
i originally had yes or no selections for judge level radio here, will remove the No from end of it